regions/shapes/ellipse.py

```diff
@@ -102,16 +102,27 @@ def to_mask(self, mode='center', subpixels=5):
         else:
             use_exact = 1
 
-        fraction = elliptical_overlap_grid(xmin, xmax, ymin, ymax, nx, ny,
-                                           self.major, self.minor,
-                                           self.angle.to(u.deg).value,
-                                           use_exact, subpixels)
+        fraction = elliptical_overlap_grid(
+            xmin, xmax, ymin, ymax, nx, ny,
+            self.major, self.minor,
+            self.angle.to(u.deg).value,
+            use_exact, subpixels,
+        )
 
         return Mask(fraction, bbox=bbox)
 
     def as_patch(self, **kwargs):
-        # TODO: needs to be implemented
-        raise NotImplementedError
+        """Matplotlib patch object for this region (`matplotlib.patches.Ellipse`).
+        """
+        from matplotlib.patches import Ellipse
+        xy = self.center.x, self.center.y
+        width = 2 * self.major
+        height = 2 * self.minor
+        # TODO: think about how to map major / minor / angle to MPL parameters.
```

Comment:
@larrybradley @astrofrog - OK if I change the argument order for ellipse to major / minor?
This seems more natural to me and is more in line with http://photutils.readthedocs.io/en/latest/api/photutils.EllipticalAperture.html .

What about the `angle` passed to MPL.
Does anyone know if it's OK as defined here?
I was looking at this for guidance:
http://photutils.readthedocs.io/en/latest/_modules/photutils/aperture/ellipse.html#EllipticalAperture.plot